blog-article.html

```diff
@@ -0,0 +1,454 @@
+                            Tony Nguyen
+                        </span>
+                        <time datetime="2022-06-20">
+                            <img src="img/calendar.svg" class="blog-data" width="16" height="16" alt="data" >
+                            20 Jun 2022
+                        </time>
+                    </div>
+                    <p>
+                        The Basilisks that players collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the <a href="#" class="content__text">Basilisks they have</a> gathered to battle other players to win ether (ETH).
+                        In Balthazar Dragons, players are immersed in a 3D open world to explore and capture dragon-like beasts called Basilisks.  Balthazar Dragons is an upcoming fantasy role-playing game developed on the Ethereum blockchain by a decentralized autonomous organization (DAO) called the Balthazar DAO.
+                    </p>
+                    <ul>
+                        <li>
+                            The Basilisks that players collect are represented with NFTs on Ethereum’s blockchain with real-world value. Players use the Basilisks they have gathered to battle other players to win ether (ETH).
+
+                        </li>
+                        <li>
+                            In Balthazar Dragons, players are immersed in a 3D open world to explore and capture dragon-like beasts called Basilisks.  Balthazar Dragons is an upcoming fantasy role-playing <a href="#" class="content__text">game developed</a>  on the Ethereum blockchain by a decentralized autonomous organization (DAO) called the Balthazar DAO.
+                        </li>
+                    </ul>
+                    <ul class="pointer-list">
+                        <li>
+                            List name 1
+                        </li>
+                        <li>
+                            List name 2
+                        </li>
+                        <li>
+                            List name 3
+                        </li>
+                        <li>
+                            List name 4
+                        </li>
+                    </ul>
+                    <div class="blockquote">
+                        <blockquote>
+                            The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers
+                        </blockquote>
+                        <cite>
+                            <span>8,5</span>/
+                            <span>10</span>
+                            Markout Corporation
+                        </cite>
+                    </div>
+                    <div>
+                        <figure>
+                            <img src="img/dragon2.jpg" width="448" height="390" alt="dragon-face">
+                        <figcaption>Photo caption</figcaption>
```

Comment:
It looks like we need to add some space here to align the code

Reply:
Done